partner_financial_risk/__manifest__.py

```diff
@@ -5,17 +5,19 @@
 {
     'name': 'Partner Financial Risk',
     'summary': 'Manage partner risk',
-    'version': '9.0.1.0.0',
+    'version': '10.0.1.0.0',
     'category': 'Sales Management',
     'license': 'AGPL-3',
     'author': 'Tecnativa, Odoo Community Association (OCA)',
     'website': 'https://www.tecnativa.com',
-    'depends': ['account'],
+    'depends': [
+        'account',
+        'sales_team',
```

Comment:
I think that this dependendy is not needed.
You can remove 'base.group_sale_manager' and keep only 'account.group_account_manager'.
That group was because depends of base.